plugins/modules/inventory_workflow_manager.py

```diff
@@ -2247,13 +2247,12 @@ def get_interface_from_id_and_name(self, device_id, interface_name):
                 interface_id = response["id"]
                 self.log("""Successfully fetched interface ID ({0}) by using device id {1} and interface name {2}."""
                          .format(interface_id, device_id, interface_name), "INFO")
-                return interface_id
+                return response
 
         except Exception as e:
-            error_message = "Error while fetching interface id for interface({0}) from Cisco Catalyst Center: {1}".format(interface_name, str(e))
-            self.log(error_message, "ERROR")
-            self.msg = error_message
             self.status = "failed"
+            self.msg = "Error while fetching interface id for interface({0}) from Cisco Catalyst Center: {1}".format(interface_name, str(e))
```

Comment:
`self.msg = "Failed to retrieve interface ID for interface({0}) from Cisco Catalyst Center: {1}".format(interface_name, str(e))
`

Reply:
updated.